collect_app/src/main/res/layout/geotrace_layout.xml

```diff
@@ -77,6 +77,14 @@
                 android:visibility="gone"
                 app:srcCompat="@drawable/ic_pause_36" />
 
+            <android.support.v7.widget.AppCompatImageButton
+                android:id="@+id/backspace"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:padding="15dp"
+                android:layout_margin="5dp"
+                android:src="@drawable/ic_backspace_arrow_selector" />
```

Comment:
Why `android:src` not `app:srcCompat` like in other buttons. it's a vector file so we should rather use `app:srcCompat`.

Reply:
Oops!  Fixed.